app/src/main/java/org/permanent/permanent/PermanentFCMService.kt

```diff
@@ -129,44 +179,50 @@ class PermanentFCMService : FirebaseMessagingService() {
     }
 
     private fun prepareNotification(
-        remoteMessage: RemoteMessage,
         record: Record,
         notificationType: FCMNotificationType
     ) {
-        if (notificationType == FCMNotificationType.SHARE_LINK_REQUEST) {
-            remoteMessage.data[FCMNotificationKey.FROM_ACCOUNT_NAME]?.let {
-                showNotification(
-                    it,
-                    getString(
-                        R.string.notification_body_share_link_request, it,
-                        remoteMessage.data[FCMNotificationKey.SHARE_NAME]
-                    ),
-                    getShareLinkViewIntent(record)
-                )
-            }
-        } else {
-            remoteMessage.data[FCMNotificationKey.INVITED_EMAIL]?.let {
-                val recordName = if (record.type == RecordType.FILE)
-                    remoteMessage.data[FCMNotificationKey.RECORD_NAME]
-                else remoteMessage.data[FCMNotificationKey.FOLDER_NAME]
-                showNotification(
-                    it,
-                    getString(R.string.notification_body_share_invitation_acceptance, it, recordName),
-                    getShareLinkViewIntent(record)
-                )
+        fcmMessage.data[FCMNotificationKey.TO_ARCHIVE_NUMBER]?.let { toArchiveNr ->
+            fcmMessage.data[FCMNotificationKey.TO_ARCHIVE_NAME]?.let { toArchiveName ->
+                if (notificationType == FCMNotificationType.SHARE_LINK_REQUEST) {
+                    fcmMessage.data[FCMNotificationKey.FROM_ACCOUNT_NAME]?.let {
+                        showNotification(
+                            it,
+                            getString(
+                                R.string.notification_body_share_link_request, it,
+                                fcmMessage.data[FCMNotificationKey.SHARE_NAME]
+                            ),
+                            getShareLinkViewIntent(record, toArchiveNr, toArchiveName)
+                        )
+                    }
+                } else { //FCMNotificationType.SHARE_INVITATION_ACCEPTANCE)
```

Comment:
is this commented out line still required?

Reply:
It's for code readability